src/providers/instagram/conversation.ts

```
@@ -0,0 +1,159 @@
+        status: null,
+        thread: null,
+        replies: null,
+        author: null,
+        cursor: null
+      }
+    };
+  }
+  const item = page.item;
+  const htmlBody = page.html;
+  const refererForGraphql =
+    page.pathUsed ?? `/p/${encodeURIComponent(shortcode)}/`;
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)